transport/src/main/java/io/netty/channel/nio/AbstractNioByteChannel.java

```diff
@@ -219,13 +219,11 @@ protected void doWrite(ChannelOutboundBuffer in) throws Exception {
                 if (done) {
                     in.remove();
                 } else {
-                    incompleteWrite(setOpWrite);
                     break;
```

Comment:
@Scottmitch yep.. that said let me add a comment why we break here and return before ... Just so we remember later on ;)